source/common/stats/stats_impl.h

```diff
@@ -412,8 +412,12 @@ class HistogramImpl : public Histogram, public MetricImpl {
 class HeapRawStatDataAllocator : public RawStatDataAllocator {
 public:
   // RawStatDataAllocator
+  typedef std::unordered_map<std::string, RawStatData*> StringRawDataMap;
   RawStatData* alloc(const std::string& name) override;
   void free(RawStatData& data) override;
+
+private:
+  StringRawDataMap stats_set_;
```

Comment:
nit: since `std::set` is a different data structure entirely, can we name this something like `stats_map_` or just `stats_`?